### PR TITLE
feat: responsiveness

### PR DESCRIPTION
**Pull request by renancaraujo**
Screenshot 2023-11-14 at 23-03-05 Flame

Screenshot 2023-11-14 at 23-03-31 Flame

Screenshot 2023-11-14 at 23-03-51 Flame

### CODE REVIEW

**Review by spydon — APPROVED** [review]
Iz broken, LGTM!

**Review by spydon — APPROVED** [review]
Iz not broken, LGTM!

File: Gemfile.lock [hunk 1]
```diff
@@ -63,6 +64,8 @@ GEM
     rexml (3.2.5)
     rouge (4.1.2)
     safe_yaml (1.0.5)
+    sass-embedded (1.64.1-x86_64-darwin)
```
Comment (luanpotter) [inline Gemfile.lock:67]: I tried using sass to have proper variables on css but failed to make it work!

File: index.html [hunk 2]
```diff
@@ -3,99 +3,108 @@
...
+        href="https://github.com/flame-engine/flame"
+        >GitHub</a
+      >
     </div>
   </div>
 
-  <div class="my-8">
-    <img src="assets/images/separator.png" />
-  </div>
+  {% include separator.html %}
```
Comment (luanpotter) [inline index.html:37]: cool! I was trying to do components but didnt had time to investigate further

File: index.html [hunk 3]
```diff
@@ -3,99 +3,108 @@
...
+      class="flex flex-col md:flex-row lg:flex-row space-y-20 md:space-y-0 lg:space-x-16 py-24"
+    >
+      {% include gamecard.html title="Watchsteroids" description="A Code
+      challenge t build a Flame game in 17 days, By the Flutter team"
+      icon="assets/icons/favicon.png" link="/" %} {% include gamecard.html
+      title="Watchsteroids" description="A Code challenge t build a Flame game
+      in 17 days, By the Flutter team" icon="assets/icons/favicon.png" link="/"
+      %} {% include gamecard.html title="Watchsteroids" description="A Code
+      challenge t build a Flame game in 17 days, By the Flutter team"
+      icon="assets/icons/favicon.png" link="/" %}
```
Comment (luanpotter) [inline index.html:58]: could we format one per line?

**Review by luanpotter — APPROVED** [review]
LGTM

File: index.html [hunk 4]
```diff
@@ -3,99 +3,108 @@
...
+>
+  <div
+    class="hero flex flex-col items-center space-y-8 py-16 md:py-18 lg:py-28"
+  >
+    <div class="flame-demo h-28 md:h-40"></div>
+
+    <p
+      class="font-bold center font-accent text-center text-3xl md:text-4xl lg:text-6xl"
+    >
       Versatile, Open Source & Runs Everywhere
```
Comment (spydon) [inline index.html:17]: Wasn't it Flexible that we agreed on? Or maybe we never settled that?

Comment (renancaraujo) [reply index.html:17]: Idk man, I am not changing content on this pr

Comment (spydon) [reply index.html:17]: Ship it, and we'll do another content overhaul PR! (If there are more content things that should be changed)

**Pull request merged**